src/dotnet/commands/dotnet-new/CSharp_MSBuild/$projectName$.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
-    <OutputPath>bin\$(Configuration)</OutputPath>
+    <OutputPath>bin\$(Configuration)\$(TargetFramework)</OutputPath>
```

Comment:
This should already be happening in the SDK.  See

https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NETCore.Build.Tasks/build/netstandard1.0/Microsoft.NETCore.TargetFrameworkInference.targets#L86

Why do we need to do it here?